### PR TITLE
Refs #22338 - add worker_timeout param

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -75,6 +75,10 @@
 #
 # $rest_client_timeout:: Timeout for Katello rest API
 #
+# $pulp_worker_timeout:: The amount of time (in seconds) before considering a worker as missing. If Pulp's
+#                        mongo database has slow I/O, then setting a higher number may resolve issues where workers are
+#                        going missing incorrectly. Defaults to 30.
```
Comment (ekohl) [inline manifests/init.pp]: The default in params.pp is 60. Not sure which we should default to but they should match :)

File: manifests/init.pp [hunk 2]
```diff
@@ -89,6 +93,7 @@
   String $qpid_interface = $::katello::params::qpid_interface,
   String $qpid_hostname = $::katello::params::qpid_hostname,
   Integer[1] $num_pulp_workers = $::katello::params::num_pulp_workers,
+  Integer[0] $pulp_worker_timeout = $katello::params::pulp_worker_timeout,
```
Comment (sean797) [inline manifests/init.pp]: Do you mind also properly namespacing this 

```
$::katello::params::pulp_worker_timeout,

**Comment by chris1984** [comment]
@ekohl can I get another review please?

**Comment by chris1984** [comment]
@ekohl can you review again when you get a moment please, tests are failing for a different reason.

**Comment by ekohl** [comment]
Looks like we need to pin rdoc in modulesync :(

**Pull request merged**